wagtail/wagtailadmin/templates/wagtailadmin/notifications/base.html

```diff
@@ -0,0 +1,132 @@
+	padding-bottom: 12px !important;
+}
+body[yahoo] .force-col-last {
+	display: block;
+	padding-right: 0 !important;
+	padding-bottom: 0 !important;
+}
+body[yahoo] .center {
+	text-align:center;
+}
+body[yahoo] .show {
+	display: block !important;
+}
+body[yahoo] .hide {
+	display: none !important;
+}
+body[yahoo] .img {
+	width: 130px !important;
+	float:left;
+}
+body[yahoo] .img img {
+	width:100% !important;
+}
+body[yahoo] .text {
+	width: 150px !important;
+	float:right;
+}
+{% endblock %}
+</style>
+</head>
+<body style="background-color:#E6E6E6;padding:0px;margin:0px;" yahoo="fix" topmargin="0" bottommargin="0" leftmargin="0" rightmargin="0">
+
+<table width="600" border="0" align="center" class="mobile" cellspacing="0" cellpadding="0">
+  <tr>
+    <td>
+    	{% block branding_logo %}<img width="100%" border="0" style="display: block;" alt="" src="{{ settings.BASE_URL }}{% static 'wagtailadmin/images/email-header.jpg' %}" />{% endblock %}
+    </td>
+  </tr>
+</table>
+<table width="600" border="0" align="center" class="mobile" cellspacing="0" cellpadding="0">
+  <tr>
+    <td style="background:#FAFAFA;"><table width="600" border="0" align="center" cellspacing="0" cellpadding="0" class="mobile">
+      <tr>
+        <td height="30" class="spacer_height">&nbsp;</td>
+      </tr>
+    </table>
+      <table width="600" border="0" align="center" cellpadding="0" cellspacing="0" class="mobile">
+        <tr>
+          <td width="30" class="spacer">&nbsp;</td>
+          <td><table width="100%" border="0" cellspacing="0" cellpadding="0" align="left" class="content">
+            <tr>
+              <td style="font-family:Arial, Helvetica, sans-serif; font-size:12px; color: #000000; line-height: 18px;">
+                {% block title %}
+                <div style="font-size:18px; line-height:24px; color: #43B1B0; font-weight:bold; display:inline-block; width: 100%; margin-bottom:10px;">{% block title_content %}{% endblock %}</div>
+                {% endblock %}
+
+                {% block greeting %}
+                <p>{% blocktrans with username=user.get_short_name|default:user.get_username %}Hello {{ username }},{% endblocktrans %}</p>
+                {% endblock %}
+
+                {% block content %}
+                {% endblock %}
+
+                <div style="font-size: 10px; margin-top: 20px;">{% trans "Edit your notification preferences here:" %} <a href="{{ settings.BASE_URL }}{% url 'wagtailadmin_account_notification_preferences' %}">{{ settings.BASE_URL }}{% url 'wagtailadmin_account_notification_preferences' %}</a></div>
+              </td>
+            </tr>
+          </table></td>
+          <td width="30" class="spacer">&nbsp;</td>
+        </tr>
+      </table>
+      <table width="600" border="0" align="center" cellspacing="0" cellpadding="0" class="mobile">
+        <tr>
+          <td height="30" class="spacer_height">&nbsp;</td>
+        </tr>
+      </table></td>
+  </tr>
+</table>
+</body>
+</html>
```

Comment:
I was talking about the empty table, the br tags could be removed so I removed them.